package.json

```diff
@@ -78,7 +78,7 @@
   },
   "homepage": "https://github.com/bbc/simorgh#readme",
   "dependencies": {
-    "@bbc/gel-foundations": "3.1.0",
+    "@bbc/gel-foundations": "3.2.0",
```

Comment:
despite the name I believe this is a psammead package, can we leave it out of this PR please